os/hal/lib/fallback/I2C/hal_i2c_lld.c

```diff
@@ -324,6 +324,18 @@ void i2c_lld_stop(I2CDriver *i2cp) {
   (void)i2cp;
 }
 
+/**
+ * @brief   Deactivates the I2C peripheral.
+ *
+ * @param[in] i2cp      pointer to the @p I2CDriver object
+ *
+ * @notapi
+ */
+void i2c_lld_soft_stop(I2CDriver *i2cp) {
+  /* just stop if there's no "soft stop option"

```

Comment:
Missed closing comment.

Reply:
shrug.  I guess none of our boards use the fallback implementation.

Fixed, thanks.